plugins/test/unit/plugins/distributors/yum/test_publish.py

```diff
@@ -155,7 +155,7 @@ def test_publish(self, mock_publish_distribution, mock_publish_rpms, mock_publis
         mock_publish_rpms.assert_called_once()
         mock_publish_drpms.assert_called_once_with(mock_publish_distribution(),
                                                    repo_content_unit_q=None)
-        mock_publish_errata.assert_called_once_with(repo_content_unit_q=None)
+        mock_publish_errata.assert_called_once()
```

Comment:
Is there a unit test path that covers the new case where we *do* want to assert `mock_publish_errata` was called once with a `repo_content_unit_q` kwarg?